src/HTTPSock.cpp

```diff
@@ -743,7 +743,7 @@ bool CHTTPSock::PrintHeader(off_t uContentLength, const CString& sContentType,
     for (const auto& it : m_msResponseCookies) {
         Write("Set-Cookie: " + it.first.Escape_n(CString::EURL) + "=" +
               it.second.Escape_n(CString::EURL) + "; HttpOnly; path=/;" +
-              (GetSSL() ? "Secure;" : "") + "\r\n");
+              (GetSSL() ? "Secure;" : "") + " SameSite=strict;\r\n");
```

Comment:
Shouldn't it be `Strict` with capital S? To follow the example from RFC draft you linked.

Reply:
Yes, thanks. Fixed